Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -4136,6 +4136,9 @@ bool CStaticFunctionDefinitions::WarpPedIntoVehicle(CPed* pPed, CVehicle* pVehic
     assert(pPed);
     assert(pVehicle);
 
+    if (pVehicle->GetMaxPassengers() == VEHICLE_PASSENGERS_UNDEFINED)
```

Comment:
```suggestion
    if (uiSeat > 0 && pVehicle->GetMaxPassengers() == VEHICLE_PASSENGERS_UNDEFINED)
```

Reply:
By checking if seat is higher than 0 we can allow warping to vehicles on driver seat even if passenger seat count is undefined.

Reply:
Right, yeah